src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -189,6 +189,21 @@ public void deleteAllStudents() {
         assertTrue(modelManager.hasModuleClass(new ModuleClass(CS2100_LAB.getName())));
     }
 
+    @Test
+    public void deleteAllModuleClasses() {
+        modelManager.addStudent(ALICE);
+        modelManager.addStudent(BENSON);
+        modelManager.addModuleClass(CS2103T_TUTORIAL);
+        modelManager.addModuleClass(CS2100_LAB);
+
+        modelManager.deleteAllModuleClasses();
+
+        assertTrue(modelManager.hasStudent(ALICE));
+        assertTrue(modelManager.hasStudent(BENSON));
+        assertFalse(modelManager.hasModuleClass(new ModuleClass(CS2103T_TUTORIAL.getName())));
+        assertFalse(modelManager.hasModuleClass(new ModuleClass(CS2100_LAB.getName())));
```

Comment:
Would it be better to test against the moduleClasses directly? Since we are adding (and removing) them along with their studentUuids.
```suggestion
        assertFalse(modelManager.hasModuleClass(CS2103T_TUTORIAL));
        assertFalse(modelManager.hasModuleClass(CS2100_LAB));
```